Telerik.JustMock/PrivateAccessor.cs

```diff
@@ -46,10 +45,11 @@ namespace Telerik.JustMock
 	/// dynamic acc = new PrivateAccessor(myobj);
 	/// acc.PrivateProperty = acc.PrivateMethod(123); // PrivateProperty and PrivateMethod are private members on myobj's type.
 	/// </remarks>
-	public sealed class PrivateAccessor : IDynamicMetaObjectProvider
+	public sealed class PrivateAccessor : PrivateAccessorBase, IDynamicMetaObjectProvider
```

Comment:
I add this comment to clearly mark inheritance from PrivateAccessorBase which I find to be deviating from "IS A" OOP design principle.

Reply:
@lyubomir-rusev, fixed with [595ded1](https://github.com/telerik/JustMockLite/pull/42/commits/595ded10a032d8e33576643110280741ea96a82d)